README.md

```diff
@@ -1,4 +1,4 @@
-# loadero-py-testui-commands
+# loadero-py-testui-commands ![PyPI - Version](https://img.shields.io/pypi/v/loadero-py-testui-commands)
```

Comment:
Should add link to https://pypi.org/project/loadero-py-testui-commands/ I think. Currently clicking on the badge just opens the image in a new tab.